packages/frontend/src/components/common/balance/helpers.js

```diff
@@ -22,6 +22,17 @@ export const showInYocto = (amountStr) => {
     return formatWithCommas(amountStr) + ' yoctoNEAR';
 };
 
+export function formatErrorBalance (msg) {
+    const regExp = /\d* yoctoNEAR/;
+    const yoctoSubString = msg.match(regExp);
+    if (yoctoSubString) {
+        const nearAmount = formatNearAmount(yoctoSubString[0].split(' ')[0]);
```

Comment:
Do we have any rounding here? Or it will be the same length:
`102671515282755412721583 yoctoNEAR` -> `0.102671515282755412721583 NEAR`

Reply:
But in general we don't even need to get to this place. We need to track such limits in our form )

Reply:
What are you talking about? This is how error looks like 
<img width="427" alt="Screenshot 2022-10-19 at 15 47 32" src="https://user-images.githubusercontent.com/7001916/196695051-69614523-a36b-49b2-9413-05f51ba1c339.png">

Reply:
We mean by default `formatNearAmount()` doesn't round the source amount

![round](https://user-images.githubusercontent.com/61930014/196705402-5c0d507d-f488-4470-8606-c837a338d039.png)